site/content/docs/api-checks/setup-teardown-scripts.md

```diff
@@ -154,6 +154,18 @@ Response properties are exposed a standard Javascript object. These are only ava
 | `response.timings`  | Various timestamps for the request stage relative to the starting time.  | Object |
 | `response.timingPhases`  | Time durations for each request phase. | Object |
 
+### General runtime variables
+
+The setup and teardown runtime also exposes a set of generic variables you can use to figure out what 
```

Comment:
May I propose to make it shorter? 😊

> The setup and teardown runtime also exposes a set of environment variables (e.g. `process.env.CHECK_NAME`) to figure out what check, check type etc. you are running.

Reply:
@stefanjudis sure go for it.

Reply:
Haha. My proposal was actually in the comment. :) I'll push and approve. :)